Lib/test/test_memoryio.py

```diff
@@ -11,6 +11,13 @@
 import pickle
 import sys
 
+class IntLike():
```

Comment:
Hmm... I found more than that, though they're all in tests. Maybe it's just my personal preference - don't worry about it if you don't want to